pkg/utils/devlink.go

```diff
@@ -0,0 +1,79 @@
+package utils
```

Comment:
I think we can make most of the functions private here right?

Reply:
I've made some of the functions private, but could not make all private.